api/institutions/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer, RelationshipField
+
+class InstitutionSerializer(JSONAPISerializer):
+    name = ser.CharField(required=True)
+    id = ser.CharField(required=True, source='_id')
+
+    nodes = RelationshipField(
+        related_view='institutions:institution-nodes',
```

Comment:
If using relationship self links to add an institution for a node, will also need a `self_view` and `self_view_kwargs`.